app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -203,32 +203,32 @@
     android:paddingTop="@dimen/spacing_8"
     android:paddingBottom="@dimen/spacing_8">
 
-    <com.google.android.material.button.MaterialButton
-      android:id="@+id/doneButton"
-      style="?attr/materialButtonStyle"
+    <org.simple.clinic.widgets.ProgressMaterialButton
```

Comment:
I don't think we need to switch to `ProgressMaterialButton` here since, we need it to be `MaterialButton`

Reply:
This has been changed in the later commit, PTAL

Reply:
Yeah but is that even necessary? 🤔 . We are adding `ProgressMaterialButton` in this commit and removing it in the next one.

Reply:
No It's not. I'll edit that commit